integration_test/third_party_apps_test/main_test.go

```diff
@@ -114,29 +114,20 @@ const (
 
 // distroFolder returns the distro family name we use in our directory hierarchy
 // inside the scripts directory.
-func distroFolder(imageSpec string) (string, error) {
-	if gce.IsWindows(imageSpec) {
+func distroFolder(vm *gce.VM) (string, error) {
+	if gce.IsWindows(vm.ImageSpec) {
```

Comment:
[Optional] Should this eventually become `gce.IsWindows(vm)`?

Reply:
Eventually, yes. Out of scope for this PR.